webapp/lib/controller.dart

```diff
@@ -96,14 +95,34 @@ void command(UIAction action, Data actionData) {
     case UIAction.signOutButtonClicked:
       platform.signOut();
       break;
+    case UIAction.viewProject:
+      project = actionData;
+      router.routeTo('#/dashboard');
+      break;
+    case UIAction.configureProject:
+      project = actionData;
+      router.routeTo('#/project-configuration');
+      break;
+    case UIAction.addProject:
+      break;
+    case UIAction.addTeamMember:
+      break;
   }
 }
 
 void loadAuthView() {
   view.contentView.renderView(new view.AuthMainView());
 }
 
+void loadProjectSelectorView() {
+  view.navView.projectTitle = '';
+  view.navView.projectOrganizations = ["Africa's Voices"];
```

Comment:
can we extract `projectOrganizations` in a constant in `model.dart`?

Reply:
Extracted it